robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             API Check And Enable Gift Entry
+Suite Teardown  Run Keywords
+...             Delete Session Records
+...             Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+
+
```

Comment:
u dont have to add this if you are not using any variables

Reply:
Removed.